input/otlp/logs.go

```diff
@@ -212,16 +213,17 @@ func (c *Consumer) convertLogRecord(
 		)
 	}
 
-	if eventDomain == "device" && eventName != "" {
+	if (eventDomain == "device" && eventName != "") || strings.HasPrefix(eventName, "device.") {
 		event.Event.Category = "device"
-		if eventName == "crash" {
+		action := strings.TrimPrefix(eventName, "device.")
```

Comment:
The goal of trimming the `device.` prefix here is so that any existing verifications down the road for specific action names, such as `crash` for example, keep working as usual. However, I'm open to alternative approaches in case this change could cause issues that I might not be aware of.